fundamentals/shapes.md

```diff
@@ -130,15 +130,15 @@ As note in the section on 3D rendering, when using 3D rendering the shapes layer
 
 All our layers can be rendered in both 2D and 3D mode, and one of our viewer buttons can toggle between each mode. The number of dimensions sliders will be 2 or 3 less then the total number of dimensions of the layer. See for example the `examples/nD_shapes.py` to see shapes in both 2D and 3D:
 
-![image](./resources/nD_shapes.gif)
+![image]({{ '/assets/tutorials/nD_shapes.gif' | relative_url }})
```

Comment:
This syntax is giving me a headache... Is part of the plan to move away from Jekyll? :joy:

Reply:
Was a headache to write too! I'd prefer not to switch from Jekyll unless you know of any alternatives that are better.

Something that we could do to fix this is by specifying a custom markdown renderer that will fix these links so that we don't have to use this gross syntax.